tools/benchmark/memory/src/main.rs

```diff
@@ -193,14 +193,15 @@ fn main() {
         let (package_name, example) = {
             // Split up the "package_name/example" string.
             let parts: Vec<&str> = package_example.split('/').collect();
-            if parts.len() != 2 {
+            if let &[first, second] = &parts[..] {
```

Comment:
Question: This is cool, but how does it work?

Reply:
slice patterns are a thing in Rust. it's just a check of the len followed by a check of the fields, though you can also use `..` in slice patterns to do open ended matches